src/style.css

```
@@ -0,0 +1,73 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf),
+    url(fonts/Avenir-Heavy.ttf);
+}
+
+:root {
+  --color-defult: #3d4e61;
+  --color-box__shadow: #5c5c5c5b;
+}
+
+body {
+  font-family: "Avenir", serif;
+  font-weight: 300;
+}
+
+.search {
+  border: none;
+  background: none;
+  outline: none;
+  padding: 0;
+  width: 100%;
+  font-size: 16px;
+}
```

font-family didn't match design
```suggestion
  outline: none;
  padding: 0;
  width: 100%;
  font-size: 16px;
  font-familiy: inherit
}
```